test/e2e-go/cli/algoh/expect/algohTimeoutTest.exp

```diff
@@ -36,63 +34,59 @@ if { [catch {
 
     puts "Successfully started node"
 
-    set timeout 5
+    # ensure Primary listing before proceeding
+    ::Algoh::WaitForFile $TEST_PRIMARY_NODE_DIR/algod-listen.net
```

Comment:
why do we need this ? When we call `goal node start`, goal is internally wait ( or should wait ) until we can connect to the health endpoint. when we do that, the file `algod-listen.net` should have already been created.

Reply:
Thinking about this, this could be the problem all along. We need to write to the `algod-listen.net` file *before* binding to the router in `server.go:207`.

Reply:
`set timeout 5` was here initially exactly because of this problem.

Reply:
done